src/Module/Conversation/Network.php

```diff
@@ -116,8 +161,13 @@ protected function content(array $request = []): string
 
 		$module = 'network';
 
-		$arr = ['query' => $this->args->getQueryString()];
-		Hook::callAll('network_content_init', $arr);
+		$hook_data = [
+			'query' => $this->args->getQueryString(),
+		];
+
+		$this->eventDispatcher->dispatch(
```

Comment:
Really a broadcast event?

Reply:
Yes, because `$arr` is no longer used in this method. Ignoring the result of `$this->eventDispatcher->dispatch()` makes this clear.